src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -83,6 +83,8 @@ protected function getPath($name)
 	 */
 	protected function parseName($name)
 	{
+		$name = strtok($name, '@');
```

Comment:
I would say using `strtok()` is a good idea. It's not a common function.
Can we use `explode($name, '@')` instead?